docs/references/ios/sign-in-with-apple.mdx

```diff
@@ -0,0 +1,105 @@
+---
+description: Learn how to use Clerk to natively Sign in with Apple.
+---
+
+# Sign in with Apple
+
+> [!WARNING]
+> The Clerk iOS SDK is currently in beta. It is **not yet recommended for production use**.
+
+This guide will teach you how to add native Sign in with Apple to your Clerk apps on Apple platforms.
+
+<Steps>
+  ### Configure the Apple social connection
+
+  To support native Sign in with Apple, you need to configure the [Apple social connection](/docs/authentication/social-connections/apple) in your Clerk dashboard.
+
+  > [!NOTE]
+  > Be sure to follow the instructions specific to native sign in with Apple flows.
+
+  ### Add the Sign in with Apple capability to your app
+
+  [Add the Sign in with Apple capability to your app](https://developer.apple.com/documentation/xcode/configuring-sign-in-with-apple#Add-the-Sign-in-with-Apple-capability-to-your-app).
+
+  ### Obtain an Apple ID Credential
+
+  To authenticate with Clerk using native Sign in with Apple in your app you need to obtain an [Apple ID Credential](https://developer.apple.com/documentation/authenticationservices/asauthorizationappleidcredential).
+  In order to obtain the credential:
+
+  1. You can use one of [Apple's built-in Sign in with Apple buttons](https://developer.apple.com/documentation/sign_in_with_apple/displaying_sign_in_with_apple_buttons_in_your_app).
+  1. You can do so manually by following the [Apple docs](https://developer.apple.com/documentation/sign_in_with_apple/implementing_user_authentication_with_sign_in_with_apple#3546457).
+
+  > [!NOTE]
+  > You must set the nonce property of the `ASAuthorizationAppleIDRequest` when requesting an Apple ID Credential in order to authenticate with Clerk.
+
+  ### Sign In / Sign Up
+
+  Once you have obtained your [Apple ID Credential](https://developer.apple.com/documentation/authenticationservices/asauthorizationappleidcredential), you can use it to authenticate with Clerk by calling `SignIn.signInWithAppleIdToken()` or `SignUp.signUpWithAppleIdToken()`
+
+  ```swift {{ filename: 'SignIn' }}
+  static func signInWithAppleIdToken(
+    idToken: String,
+    code: String? = nil
+  ) async throws -> ExternalAuthResult?
+  ```
+
+  ```swift {{ filename: 'SignUp' }}
+  static func signUpWithAppleIdToken(
+    idToken: String,
+    code: String? = nil,
+    firstName: String? = nil,
+    lastName: String? = nil,
+    captchaToken: String? = nil
+  ) async throws -> ExternalAuthResult?
+  ```
```

Comment:
iOS doesn't have a dedicated Sign In page though, and it might never since the SDK reference docs are autogenerated [here](https://swiftpackageindex.com/clerk/clerk-ios/0.34.1/documentation/clerksdk/signin) once every 24 hours (This will eventually be hosted on our site and not swiftpackageindex.com). The iOS SDK is completely separate and doesn't use the javascript core library. 

I thought these were useful callouts for the guide, and less so as a source of truth.

Reply:
Would it be better if I added links to `SignIn.signInWithAppleIdToken()` and `SignUp.signUpWithAppleIdToken()` that link to the reference?